pages/common/mkdir.md

```diff
@@ -9,3 +9,7 @@
 - Create directories recursively (useful for creating nested dirs):
 
 `mkdir -p {{path}}`
+
+- Create multiple directories recursively (note the curly braces, marking directory set):
+
+`mkdir -p dir1/dir2/{dir3/{dir4,dir5},dir6,dir7}`
```

Comment:
Perhaps the previous example should include something resembling a real path with mutiple components, e.g. `{{path/to/directory}}`